src/ResourceManager/Websites/AzureRM.Websites.Netcore.psd1

```diff
@@ -9,7 +9,7 @@
 @{
 
 # Script module or binary module file associated with this manifest.
-# RootModule = ''
+RootModule = '.\AzureRM.Websites.Netcore.psm1'
```

Comment:
I don't think this is the way we want to do it - when I asked Mark he said this would make debugging more difficult.  I'll try to figure out a better way to do this in the build.proj file

Reply:
@maddieclayton and @markcowl, what are the debug issues?  Maybe it's Netcore, but I can debug just fine the way it is.  What problems does this cause on desktop?

Also, this makes dynamic completers work without having to do a bunch of changes/fixes to UpdateModules and PublishModules.

Reply:
@twitchax see my new PR - this should fix the issue (I also added you as a reviewer)